x-pack/plugins/observability_solution/investigate_app/public/pages/details/components/investigation_timeline/investigation_timeline_filter_bar/investigation_timeline_filter_bar.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { EuiDatePicker, EuiDatePickerRange, EuiFlexGroup } from '@elastic/eui';
+import { css } from '@emotion/react';
+import moment from 'moment';
+import React from 'react';
+import { useInvestigation } from '../../../contexts/investigation_context';
+import { InvestigationEventTypesFilter } from './investigation_event_types_filter';
+
+interface Props {
+  onEventTypesSelected: (eventTypes: string[]) => void;
+}
+
+export function InvestigationTimelineFilterBar({ onEventTypesSelected }: Props) {
```

Comment:
💬  Simplified (and renamed) the timeline filter bar: replaced uiSearch with dateRange eui component and moved the update function into here.